sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -61,6 +61,7 @@ enum {
 #define SOF_SSP_PORT(x)		(((x) & GENMASK(5, 0)) << 5)
 #define SOF_SSP_GET_PORT(quirk)	(((quirk) >> 5) & GENMASK(5, 0))
 #define SOF_RT715_DAI_ID_FIX		BIT(11)
+#define SOF_SDW_MERGE_SPK_DAI		BIT(12)
```

Comment:
SOF_SDW_SPK_MULTI_DAI ?

Reply:
multi-cpu DAI in one BE DAI, not multi-BE DAI.